disnake/ext/commands/interaction_bot_base.py

```diff
@@ -991,7 +991,7 @@ async def on_message_command_error(
 
     def add_app_command_check(
```

Comment:
nit: the docstring here still mentions `.check` and `.check_once`, which isn't entirely correct

Reply:
ccd95a5a